dodola/core.py

```diff
@@ -589,7 +589,11 @@ def non_polar_dtr_ceiling(ds, ceiling):
 
     """
 
-    ds_corrected = ds.where((ds <= ceiling) & (ds.lat > -60) & (ds.lat < 60), ceiling)
+    ds_corrected = ds.where(
+        ((ds <= ceiling) & (ds["lat"] > -60) & (ds["lat"] < 60))
+        | (ds["lat"] <= -60 or ds["lat"] >= 60),
+        ceiling,
+    )
```

Comment:
This is a bit hard to parse, would something simpler like
```python
ds_corrected = ds.where((ds <= ceiling) | (ds["lat"] <= -60 or ds["lat"] >= 60), ceiling)
```
still work?

Reply:
Yes I think that works and it's better. Thanks !